sycl/doc/extensions/experimental/sycl_ext_oneapi_memory_export.asciidoc

```diff
@@ -0,0 +1,338 @@
+as having the ability to later export that memory to an external memory resource
+handle.
+
+Memory allocated through this function must only be freed using
+`free_exportable_mem`. Using `sycl::free` to deallocate memory allocated with
+this function results in undefined behavior.
+
+With the exception of the `sycl::free` function from the core SYCL
+specification, pointers to memory allocated through this function may be passed
+to any core SYCL specification API accepting device USM memory pointers.
+
+Memory allocated through this function is only available on device.
+
+Memory allocated through this function has a linear memory layout on the device 
+(which is the same as memory allocated by other USM allocation functions like 
+`sycl::malloc_device`).
+
+Zero or more properties can be passed in the `propList` parameter via an
+instance of `sycl::property_list`. Currently, this extension does not define
+any properties that can be used with this function, so the `propList` parameter
+is ignored and reserved for future use.
+
+If an invalid `exportMemHandleType` is passed, the function will throw a
+`sycl::exception` with the `errc::invalid` code.
+
+The passed `exportMemHandleType` must be supported by the operating system
+running the SYCL application. Passing an `exportMemHandleType` that is not
+supported by the operating system running the SYCL application results in
+undefined behavior.
+
+If any error occurs while allocating the memory, the function will throw a
+`sycl::exception` with the `errc::runtime` code.
+
+If the SYCL backend does not support exporting memory, the function will throw a
+`sycl::exception` with the `errc::feature_not_supported` code.
+
+```c++
+
+namespace sycl::ext::oneapi::experimental {
+
+template <export_external_mem_handle_type ExportMemHandleType>
+exported_mem_t<ExportMemHandleType>
+export_device_mem_handle(void *deviceMemory, const sycl::device &syclDevice,
+                         const sycl::context &syclContext);
+
+template <export_external_mem_handle_type ExportMemHandleType>
+exported_mem_t<ExportMemHandleType>
+export_device_mem_handle(void *deviceMemory, const sycl::queue &syclQueue);
+
+}
+```
+
+The `export_device_mem_handle` function accepts a `void *` representing a device
+allocation made using `alloc_exportable_device_mem`.
+
+The return type is determined by the template parameter,
+`ExportMemHandleType`.
+
+The value of `ExportMemHandleType` must match the value passed to
+`alloc_exportable_device_mem` when the memory was allocated.
+
+The returned handle can be used to import the SYCL allocated memory into an
+external API, such as Vulkan or DirectX.
+
+If any error occurs while exporting the memory handle, the function will throw a
+`sycl::exception` with the `errc::runtime` code.
+
+If the SYCL backend does not support exporting memory, the function will throw a
+`sycl::exception` with the `errc::feature_not_supported` code.
```

Comment:
There's no need to say this because you can't get a `deviceMemory` pointer if the device doesn't support exporting memory.  If you want to keep this error, reword using the aspect as I mentioned above.